package NGeoDB;

enum ETranslocalityHack {
    NO_HACK     = 0;
    CRIMEA_HACK = 1;
}
